### ISSUE TITLE
Using NAudio in C# 2.0 VS 2005 application

### ISSUE THREAD

**Issue by saurabh-bagchi-mindteck** [naudio/NAudio#638]
Hi,

Is it possible to implement to play audio files using NAudio in C# 2.0 VS 2005 application? How to get a sample application where NAudio is used to implement using C# 2.0 in a VS 2005 platform? 

Thanks in advance for any support received.

Regards,
Saurabh

**Comment by WinformsWizard** [comment]
I believe NAudio works with .NET Standard 2.0 or greater. I may be wrong, but I do not think it works with C# 2.0.

**Comment by markheath** [comment]
Yes, closing this as NAudio cannot be used from Visual Studio 2005 / C# 2.0. NAudio 2.x targets .NET Framework 4.x and .NET Standard 2.0, and the upcoming NAudio 3 (in development on `main`) requires .NET 9+. None of these are reachable from VS 2005, which tops out at .NET 3.5 and does not support the language and runtime features modern NAudio depends on (generics-heavy APIs, `Span<T>`, async/await, source generators, etc.). To use NAudio you will need to move to a current Visual Studio (2026 or later) and a modern .NET target.
